### PR TITLE
[5.6] Use Arr::wrap

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: src/Illuminate/Foundation/Console/Kernel.php [hunk 1]
```diff
@@ -195,7 +196,7 @@ public function command($signature, Closure $callback)
      */
     protected function load($paths)
     {
-        $paths = array_unique(is_array($paths) ? $paths : (array) $paths);
+        $paths = array_unique(Arr::wrap($paths));
```
Comment (sisve) [inline src/Illuminate/Foundation/Console/Kernel.php]: This is a change of behavior if $paths is an object.

File: src/Illuminate/Support/Str.php [hunk 2]
```diff
@@ -153,7 +153,7 @@ public static function finish($value, $cap)
      */
     public static function is($pattern, $value)
     {
-        $patterns = is_array($pattern) ? $pattern : (array) $pattern;
+        $patterns = Arr::wrap($pattern);
```
Comment (sisve) [inline src/Illuminate/Support/Str.php]: This is a change of behavior if $paths is an object.

**Comment by sisve** [comment]
@carusogabriel Your thumbs-down reactions are unclear. Could you clarify? Do you disagree with my comments?

**Comment by carusogabriel** [comment]
@sisve Yes, I do disagree. Here are some reasons:
- Casting and wrapping have the same behavior: https://3v4l.org/QNoFE
- This was already done previously, with your same comment: https://github.com/laravel/framework/pull/21707
- The phpDocs says: `array|string`
- If this would be a change of behavior, the tests wouldn't pass

Thanks.

**Pull request merged**